src/java-runtime/assets/jdk.installation.tsx

```diff
@@ -148,6 +148,7 @@ export class JdkInstallationPanel extends React.Component<JdkInstallationPanelPr
           </div>
           <div className="tab-pane fade" id="otherJdkPanel" role="tabpanel" aria-labelledby="otherJdkTab">
             <ul id="jdkProviderList" className="list-unstyled">
+              <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.microsoft.com%2Fopenjdk%22" title="Microsoft Build of OpenJDK">Microsoft Build of OpenJDK</a><sup className="newtag">NEW</sup></li>
```

Comment:
Directly use `Badge` from bootstrap? https://getbootstrap.com/docs/5.0/components/badge/

Reply:
No. Currently it's exactly what I want.